crates/iota-sdk-types/src/crypto/intent.rs

```diff
@@ -21,7 +33,8 @@
 /// ```text
 /// intent = intent-scope intent-version intent-app-id
 /// ```
```

Comment:
Can't comment there but please change L23 to `IntentAppId`, also check if there is more